packages/config/src/utils/utils.ts

```diff
@@ -156,7 +190,8 @@ export const getContracts = (
     mergeFlowJSONs,
     mergePipe(
       mapDeploymentsToNetworkAddress(network),
```

Comment:
We also need to map `dependencies` deployments to their network address (i.e. when user deploys a dependency on emulator, it will not have an alias but may will have a deployment to an account, so FCL needs to be able to resolve this.)

i.e.

```json
{
    "dependencies": {
        "Foobar": {
            ...,
            "aliases": {
                "testnet": ...
                "mainnet": ...
            }
        }
    },
    "accounts": {
        "emulator: {
            ...
            "address": "0x01"
        }
    },
    "deployments": {
        "emulator": {
           "emulator-account": ["Foobar"]
        }
    }
}
```

On the emulator network,`Foobar` would be mapped to `0x01` because it is listed as a deployment.

Reply:
Hmm, I'm not sure which part needs to be modified.   
The current code still includes parsing for deployments.  
I just added parsing for `dependencies` after parsing `contracts`, similar to how it's done.

Currently, the priority order for pipe overriding is:

`dependencies` > `contracts` > `deployments`

Are you saying that I need to adjust the `override` priority or do I need to handle the `emulator` network in a specialized way?

Reply:
Ah, yeah sorry I made a mistake, this will handle the deployments properly.  I thought that special behaviour was required for `mapDeploymentsToNetworkAddress` for contract deployments that came from dependencies but this will work.

My bad 😅